streams/src/main/java/org/apache/kafka/streams/processor/internals/ActiveTaskCreator.java

```diff
@@ -115,7 +115,7 @@ public void reInitializeThreadProducer() {
 
     StreamsProducer streamsProducerForTask(final TaskId taskId) {
         if (processingMode != EXACTLY_ONCE_ALPHA) {
-            throw new IllegalStateException("Producer per thread is used.");
+            throw new IllegalStateException("Expected EXACTLY_ONCE to be enabled, but the processing mode was " + processingMode);
```

Comment:
nit: `EXACTLY_ONCE` -> `eos-v1` (to align to `eos-v2` below)

Reply:
I'll do it the other way around to be consistent with how we print the `processingMode` (ie change `eos-v2` to `EXACTLY_ONCE_V2` below instead)